src/features/workspace/components/__tests__/workspace-custom-instructions.test.tsx

```diff
@@ -26,9 +27,12 @@ const renderComponent = () =>
 
 test("can update custom instructions", async () => {
   server.use(
-    http.get("*/api/v1/workspaces/:name/custom-instructions", () => {
-      return HttpResponse.json({ prompt: "initial prompt from server" });
-    }),
+    http.get(
```

Comment:
Not a required change for this PR, but I am kinda wondering if we could even make a single wrapper function that can be used like this:

```tsx
mockEndpoint("..etc...", () => {
   return HttpResponse.json(...etc...)
})
```